src/test/java/org/fundacionjala/coding/kevincristhian/CesarCryptoTest.java

```diff
@@ -0,0 +1,50 @@
+package org.fundacionjala.coding.kevincristhian;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * test.
+ */
+public class CesarCryptoTest {
+
+    private static final String NEGATIVE_THREE = "-3";
+    private static final String NEGATIVE_FIVE = "-5";
+    private static final String THREE = "3";
+    private static final String FIFTY_TWO = "-52";
+    private static final String ZERO = "0";
+    private static final String FOX = "ZORRO";
+    private static final String ONE = "1";
+    private static final String FORTY = "40";
+    private static final String NEGATIVE_FORTY = "-40";
+
+    /**
+     * decode encode cesar tests.
+     */
+    @Test
+    public void decode() {
+        CesarCrypto cesarCrypto = new CesarCrypto();
```

Comment:
You are repeating this line of code in all your tests
Please move it to a @before method.